src/target/llvm/llvm_common.cc

```diff
@@ -60,10 +61,11 @@ void InitializeLLVM() {
 }
 
 void ParseLLVMTargetOptions(const Target& target, std::string* triple, std::string* mcpu,
-                            std::string* mattr, llvm::TargetOptions* options) {
```

Comment:
Please pull out the llvm target changes to a separate PR .

Reply:
done